client/tsconfig.json

```diff
@@ -1,11 +1,7 @@
 {
   "compilerOptions": {
     "target": "es5",
-    "lib": [
```

Comment:
I normally leave it on multiple lines for config files, better readability but just a thought!

Reply:
This is probably done by the linter